app/topics/show-topic/show-topic.component.html

```diff
@@ -120,4 +120,7 @@ <h3 md-line>{{ subtopic.title }}</h3>
   <md-tab *ngIf="userCanEditContent" label="{{ 'attachments' | translate }}">
     <hip-manage-attachments></hip-manage-attachments>
   </md-tab>
-</md-tab-group>
+  <md-tab *ngIf="userCanEditReview" label="{{ 'reviews' | translate }}">
```

Comment:
Reviews should be visible for everyone assigned to the topic (students and supervisors), not reviewers only. Only review submission is restricted to the reviewers.